docs/en/Running-Alluxio-on-a-Cluster.md

```diff
@@ -25,12 +25,21 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 ```
 
 Update `alluxio.master.hostname` in `conf/alluxio-site.properties` to the hostname of the machine
-you plan to run Alluxio Master on. Add the IP addresses of all the worker nodes to the
-`conf/workers` file. Finally, sync all the information to worker nodes. You can use
+you plan to run Alluxio Master on. Add the IP address or hostname of all the worker nodes to the
+`conf/workers` file. You cannot use local file system as Alluxio's under storage system if there
```

Comment:
the limitation is that you cannot use local file system when there are multiple nodes in the cluster. Even with one worker, it will still be a problem if the worker is on a different node from the master

Reply:
Done.